docs/concepts.md

```diff
@@ -0,0 +1,39 @@
+# Meltano Cloud Concepts
+
+## Meltano Cloud Users
+
+Your Meltano Cloud user account is associated with a GitHub account for single-sign-on. If you do not have a GitHub account, you can create a free account at [github.com](https://github.com).
+
+> **Note**
+>
+> During the Private and Public Beta phases, we will be collecting feedback from users regarding the login and account creation experience overall. Post-GA, we plan to offer SAML as a premium feature for Enterprise users. If you have a use case to support alternate forms of login besides GitHub auth and SAML (e.g. GitLab, or user/password auth), please let your account manager know or log an issue in our [issue tracker](https://github.com/meltano/cloud-docs/issues).
+
+## Meltano Cloud Organizations
```

Comment:
The more I think about it, the more I like organizations. It's pretty clear what it represents and there's lots of prior art for it (https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy)